src/EditorFeatures/Test2/GoToHelpers/GoToHelpers.vb

```diff
@@ -40,6 +44,30 @@ Friend Class GoToHelpers
                     Assert.True(actual.CompareTo(expected) = 0,
                                 $"Expected: ({expected}) but got: ({actual})")
                 Next
+
+                Dim actualDefintionsWithoutSpans = context.GetDefinitions().
+                Where(Function(d) d.SourceSpans.IsDefaultOrEmpty).
```

Comment:
Indenting is off here, and usually we always do a _ at the end of a previous line here so the . stays on the next line.